css/at-rules/font-face.json

```diff
@@ -277,6 +277,103 @@
             }
           }
         },
+        "font_feature_settings": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/font-feature-settings",
```

Comment:
You mixed font-feature-settings property and the font-feature-settings descriptor. Here we deal with the descriptor that is not at this URL (It is not documented on MDN).

The descriptor is only supported by Firefox, so you need to remove the mdn_url entry and put everything but firefox entries to false.